### DISCUSSION TITLE
Draw anything in pdf

### DISCUSSION THREAD

**Discussion by rdhhauzan** [wojtekmaj/react-pdf#1537] [Q&A]
I'm new to this library, can i use this library to make the user can draw anything inside pdf? because if i can i want to use this library to make a Digital Signature application

Thank you

**Comment by wojtekmaj** [comment]
Canvas is a canvas, you can do anything you want with it once it's rendered. Old example I've found:

https://codesandbox.io/s/react-pdf-watermark-5wylx

**Reply by aman2810khan** [reply]
One concern , how can we get the new blob of this updated PDF since we want to show the watermark in print preview and downloading

**Reply by wojtekmaj** [reply]
You can't, because you're not modifying PDF in any way, you just draw on the same canvas PDF.js used.
